### ISSUE TITLE
[Optional]: Run a testnet on DO (200 node test) as smoke test on this branch

### ISSUE THREAD

**Issue by sergio-mena** [cometbft/cometbft#547]

**Comment by sergio-mena** [comment]
Thanks @jmalicevic, I think they are correlated with the rounds plot (which is missing). I'm including it below.
But then again, the peaks existed in `v0.37.0-alpha.3`. More generally, (thinking of planning) we might want to allocate some time to investigate why we jump to round 1 (sometimes even round 2) so often, although my guess is that, when we optimize the mempool and the proposal, it will stoop happening.
Here are the round plots (first `v0.37.0-alpha.3` and then `sergio/test-vote-extensions-on-do`:

![rounds](https://user-images.githubusercontent.com/15381133/226552399-a63b4792-f458-49e5-ba47-6d4e2e01ea96.png)
![rounds](https://user-images.githubusercontent.com/15381133/226552497-da1d6954-cdbb-4825-b048-974ea7203bf0.png)

**Comment by adizere** [comment]
Great work Sergio, agree with your conclusions. +1 to Jasmina's questions above!

I have a couple of ideas to further improve, but should not block here:
- might be easier to read & compare plots if we ensure all comparable plots have the same Y-axis.
- in terms of X-axis, seems like most plots have ~2 minutes duration, except for the latency plots which are ~80-100 seconds.

**Comment by sergio-mena** [comment]
@adizere, you're totally right. We'll take your suggestions into account when carrying out the official QA process in a few days

**Comment by jmalicevic** [comment]
Although on second thought, it seems more blocks are created by some validators when the rounds are higher... In any case, those are outliers, nothing preventing us from merging the branches..
